### PR TITLE
add readme for popup toolkit component

### PR DESCRIPTION
**Pull request by puneet-pdx**
Changes to add documentation for Popup toolkit component

### CODE REVIEW

File: Documentation/Popup/README.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# PopupView
+
+The PopupView shows the Popup's attribute list in a Recyclerview, allows user to edit Popup's attribute list and delete the Popup.
```
Comment (gunt0001) [inline Documentation/Popup/README.md:3]:
```suggestion
The PopupView shows a Popup's attribute list in a Recyclerview, allows users to edit a Popup's attribute list and delete the Popup. 
```

File: Documentation/Popup/README.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+
+## Workflow 
+
+The app can define the `PopupView` in its view hierarchy, as `PopupView` extends FrameLayout. The app 
+has to set the `popup` and `popupManager` property on the `PopupView` with the Popup it wasnts to view/edit. The `PopupView` toolkit component
+adhers to the `MVVM` pattern and comes with the `PopupViewModel` that exposes the functionality to delete the `GeoElement`, set the edit mode on
+the `PopupView` and other LiveData events that the app can listen to, to provide user with UI notifications.
+
+Here is an example XML code that shows the PopupView and how the PopupView Properties are tied to the PopupViewModel:
```
Comment (gunt0001) [inline Documentation/Popup/README.md:13]:
```suggestion
Here is an example XML layout that shows the PopupView and how its properties are bound to the PopupViewModel:
```

File: Documentation/Popup/README.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+     popupViewModel.showSavingProgressEvent.observeEvent(viewLifecycleOwner) { isShowProgressBar ->
+            if (isShowProgressBar) {
+                requireActivity().window.setFlags(
+                    WindowManager.LayoutParams.FLAG_NOT_TOUCHABLE,
+                    WindowManager.LayoutParams.FLAG_NOT_TOUCHABLE)
+                progressBarLayout.visibility = View.VISIBLE
+            } else {
+                progressBarLayout.visibility = View.GONE
+                requireActivity().window.clearFlags(WindowManager.LayoutParams.FLAG_NOT_TOUCHABLE)
+            }
```
Comment (gunt0001) [inline Documentation/Popup/README.md:124]: For simplicity I suggest...
```suggestion
            if (isShowProgressBar) {
                showProgressBar()
            } else {
                hideProgressBar()
            }
```

**Review by gunt0001 — APPROVED** [review]
@puneet-pdx - looks good, a few minor suggestions.

File: Documentation/Popup/README.md (hunk 1 above)
Comment (puneet-pdx) [reply Documentation/Popup/README.md:3]: updated

File: Documentation/Popup/README.md (hunk 2 above)
Comment (puneet-pdx) [reply Documentation/Popup/README.md:13]: updated

File: Documentation/Popup/README.md (hunk 3 above)
Comment (puneet-pdx) [reply Documentation/Popup/README.md:124]: updated

**Pull request merged**